normandy/bundler/models.py

```diff
@@ -21,7 +21,10 @@ def ids(self):
 
     @property
     def hash(self):
-        recipe_hashes = '_'.join([recipe.content_hash for recipe in self.recipes])
+        hash_parts = []
+        for recipe in self.recipes:
+            hash_parts.extend(action.implementation_hash for action in recipe.actions.all())
```

Comment:
This is all going away soon, I guess. This was mostly just getting it to stop erroring out. Probably better to leave it as is an make it better once we implement the newer API.